app/assets/stylesheets/zeal-initiative-tracker.webflow.css

```diff
@@ -476,5 +480,92 @@ h5 {
 .icon-target {
   width: 30px;
   height: 30px;
-  margin-right: 13px;
+  margin-right: 13px
```

Comment:
It looks like the semicolon got removed here.

Reply:
I added a semicolon to line 483